webpack/webpack5-localization-plugin/src/interfaces.ts

```diff
@@ -144,6 +144,11 @@ export interface ILocalizationPluginOptions {
    * runtimeLocaleExpression produces the same output as formatLocaleForFilename.
    */
   formatLocaleForFilename?: (locale: string) => string;
+
+  /**
+   * If set to true, update usages of [contenthash] to use the true hash of the file contents
+   */
+  useTrueHashes?: boolean;
```

Comment:
Should we call this `realContentHash` to match the webpack config option of the same name that does the same thing?